### DISCUSSION TITLE
Add a * to an appropriate field of update

### DISCUSSION THREAD

**Discussion by gjf** [valinet/ExplorerPatcher#377] [General]
As manual update actually restarts explorer.exe, I think it should be marked:
![Screenshot 2021-11-16 013433](https://user-images.githubusercontent.com/322186/141863643-328358f8-a332-4ea0-beff-9211c78ccb0b.png)

**Comment by valinet** [comment]
First of all, this is more appropriate as a discussion. I don't really think it is an issue. Let me explain:

* I think it is pretty explanatory that an update will restart Explorer, like how else can it update it (technically, there are ways, not going to deny that, but hard to pull it off right in this case)
* Update does not have a `*` because it *does not require restarting File Explorer*, but rather, it actually *restarts Explorer automatically* for you in order to have the update take effect.

So yeah, a * does not really fit imo. ** is taken in some other submenu, so yeah, *** with an additional note...? Write in some brackets that it also restarts Explorer...? Does it really have to be that verbose? Isn't this pedantic, idk... Like, I can add it in, no problem...

Idk, I'd change the text "Settings marked with an (\*) require restarting File Explorer." to "Applying the settings marked with (\*) require restarting File Explorer".

Thanks, idk, I will update something about it eventually, if it's that big of a problem...
